.github/stale.yml

```diff
@@ -0,0 +1,39 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale. Set to `[]` to disable
+# We don't close any issue that is an enhancement or confirmed bug, but issues
+# waiting for reproduction cases and questions tend to get outdated.
+exemptLabels:
+  - "enhancement"
+  - "bug"
+  - "thinking"
+  - "docs"
+
+# Label to use when marking as stale
+staleLabel: "waiting-reply"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: |
+  Hey there,
+  We wanted to check in on this request since it has been inactive for at least 60 days.
```

Comment:
just thinking if i got this in my email for an issue i created.. I think moving the 'Have you reviewed the forums...' sentence to the bottom might be better. otherwise it's sort of like, "hey, you created this issue. we have done nothing in the last 60 days to help you. but you should have done more - did you read the forums??!?!" (exaggerated)

also being more explicit with the "please feel free to let us know" by saying "please reply with a comment" or something..

and being explicit that this is a bot..

and perhaps calling it an "issue" and not a "request" ? at least until we want to add pull requests?

all together something like:

```
Hey there (bot here),
We wanted to check in on this issue since it has been inactive for at least 60 days. 
If you think this is still an important issue in the latest version of [Consul](https://github.com/hashicorp/consul/compare/) or [its documentation](https://www.consul.io/docs) please reply with a comment here to let us know and we'll keep it open for investigation. 
If there is still no activity on this issue in 30 days, we will go ahead and close it.

You can peruse the [community forum](https://discuss.hashicorp.com/c/consul) too for possible tips related to your issue.
Thank you!
```

(feel free to rephrase as you see fit, of course)

Reply:
Hey! I really appreciate this suggestion, it has a great flow! I tweaked it a little bit, but stole most of it. :)